app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -93,7 +86,7 @@ fun TopToolBar() {
                     )
             )
             Text(
-                text = "RentMinder", fontSize = 25.sp, fontWeight = FontWeight.Bold
+                text = stringResource(R.string.app_name), fontSize = 25.sp, fontWeight = FontWeight.Bold
```

Comment:
Use string.xml to reference string instead of hardcoding string, allowing for more supported languages and the ability to reuse strings.

https://developer.android.com/guide/topics/resources/string-resource#String

Reply:
Good.